templates/sectionsummary.html

```diff
@@ -37,10 +39,8 @@ <h2>{{summary.title}}</h2>
               {{ summary.empty_list_text }}
             </div>
           {% endif %}
-          {% if summary.add_link %}
```

Comment:
The add link was optional in the previous version, does the new version need to handle this?

Reply:
This is removed now👍